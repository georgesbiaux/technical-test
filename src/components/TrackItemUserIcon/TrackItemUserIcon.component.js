import styled from 'styled-components';

const TrackItemUserIcon = styled.div`
  width: 60px;
  height: 60px;
  margin: 10px;
  border-radius: 50%;
  background-image: url('${props => props.pictureUrl}');
  background-size: cover;
`;

export default TrackItemUserIcon;
