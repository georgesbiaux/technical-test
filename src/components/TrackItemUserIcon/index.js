export { default } from './TrackItemUserIcon.component';
