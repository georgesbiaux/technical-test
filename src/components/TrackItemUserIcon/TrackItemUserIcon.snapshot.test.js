import React from 'react';
import { render } from 'enzyme';
import toJSON from 'enzyme-to-json';
import TrackItemUserIcon from './TrackItemUserIcon.component';

describe('<TrackItemUserIcon>', () => {
  it('should render correctly', () => {
    const props = {
      pictureUrl:
        'https://i.scdn.co/image/adcf79a6adae4e1a44d0e34860cc103e28ee4372',
    };
    const tree = render(<TrackItemUserIcon {...props} />);
    expect(toJSON(tree)).toMatchSnapshot();
  });
});
