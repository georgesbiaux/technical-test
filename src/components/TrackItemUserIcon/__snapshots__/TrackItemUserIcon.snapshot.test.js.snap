// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrackItemUserIcon> should render correctly 1`] = `
<div
  class="sc-bdVaJa dGEhtb"
>
  
</div>
`;
